code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -206,6 +204,7 @@ GLOBAL_LIST_EMPTY(roundstart_station_closets)
 
 /obj/structure/closet/Destroy()
 	id_card = null
+	REMOVE_TRAIT(src, TRAIT_STRONGPULL, null)
```

Comment:
`null` source? I don't think this does anything

In fact why remove a trait in `destroy` anyways? Usually harmless